src/media/local-display-track.ts

```diff
@@ -1,6 +1,33 @@
 import { LocalTrack } from './local-track';
 
+/**
+ * See https://developer.mozilla.org/en-US/docs/Web/API/MediaStreamTrack/contentHint.
+ */
+export type VideoContentHint = 'motion' | 'detail';
+
 /**
  * Represents a local track for a display source.
  */
-export class LocalDisplayTrack extends LocalTrack {}
+export class LocalDisplayTrack extends LocalTrack {
+  protected _videoContentHint?: VideoContentHint;
+
+  /**
+   * Create a LocalDisplayTrack from the given values.
+   *
+   * @param stream - The MediaStream for this track.
+   * @param videoContentHint - An optional content hint, describing the content of the track.
+   */
+  constructor(stream: MediaStream, videoContentHint?: VideoContentHint) {
+    super(stream);
+    this._videoContentHint = videoContentHint;
```

Comment:
shouldn't we be also setting that content hint on the actual underlying webrtc track?

Reply:
Ha, yes fair point.  Obviously we're mainly using this setting to signal to the backend, but might as well.

Reply:
done